source/frontend/src/mocks/msw.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  getMockAddresses,
+  getMockLandChars,
+  getMockLegalDescriptions,
+  getMockValues,
+} from 'mocks/bcAssessmentMock';
+// src/mocks.js
+// 1. Import the library.
+import { rest, setupWorker } from 'msw';
+
+import { getMockDescription, getMockSales } from './bcAssessmentMock';
+
+// 2. Describe network behavior with request handlers.
+export const worker = setupWorker(
+  rest.get('https://delivery.apps.gov.bc.ca/ext/sgw/geo.bca', (req, res, ctx) => {
+    if (req.url.search.includes('BCA_FOLIO_ADDRESSES_SV')) {
+      return res(ctx.delay(500), ctx.status(200), ctx.json(getMockAddresses()));
+    } else if (req.url.search.includes('BCA_FOLIO_LEGAL_DESCRIPTS_SV')) {
+      return res(ctx.delay(500), ctx.status(200), ctx.json(getMockLegalDescriptions()));
+    } else if (req.url.search.includes('BCA_FOLIO_GNRL_PROP_VALUES_SV')) {
+      return res(ctx.delay(500), ctx.status(200), ctx.json(getMockValues()));
+    } else if (req.url.search.includes('BCA_FOLIO_SALES_SV')) {
+      return res(ctx.delay(500), ctx.status(200), ctx.json(getMockSales()));
+    } else if (req.url.search.includes('BCA_FOLIO_DESCRIPTIONS_SV')) {
+      return res(ctx.delay(500), ctx.status(200), ctx.json(getMockDescription()));
+    } else if (req.url.search.includes('BCA_FOLIO_LAND_CHARS_SV')) {
+      return res(ctx.delay(500), ctx.status(200), ctx.json(getMockLandChars()));
+    }
```

Comment:
We will need to do the same setup for jest tests because jest runs in Node and there we don't have service workers. 
Ideally we would split the setupWorker/setupServer calls from the handlers to be able to reuse the URL mocking in both the browser APP and jest.

Please see this sample repo: https://github.com/mswjs/examples/tree/master/examples/rest-react/src/mocks

![image](https://user-images.githubusercontent.com/24322224/219185558-d44b6f0b-1f22-4538-857e-3d5d47b1b3c1.png)

Reply:
We would also need to update **setupTests.ts** to launch msw in jest. Like this: https://github.com/mswjs/examples/blob/master/examples/rest-react/src/setupTests.js

Reply:
True, but I think that is outside the scope of this task: https://jira.th.gov.bc.ca/browse/PSP-5496 (setup the mock to replace existing bc assessment mock).

Reply:
fair enough, then we need a tech-debt task to add msw to jest. We will benefit from having the same mocking behaviour in tests as local dev